config/pipeline.yaml

```diff
@@ -150,6 +150,12 @@ device_types:
     boot_method: qemu
     mach: qemu
 
+  minnowboard-turbot-E3826:
+    base_name: minnowboard-turbot-E3826
```

Comment:
TODO: adjust `base_name` to reflect job parameters (will be used as a TestJob name), e.g. `baseline-x86`

Reply:
I think you can just drop `base_name` here if the device type entry in YAML matches the LAVA device type name.

Reply:
Got it, thanks (they do match, removed)!

I did a [quick search for the `base_name`](https://github.com/search?q=org%3Akernelci+base_name&type=code) and I wonder if it won't affect [setting parameters for running a test](https://github.com/kernelci/kernelci-core/blob/77c16d940dfa26d738973c3cd8a74b9cd927d46c/kernelci/test.py#L166) but I might be still missing some configuration step.